lib/core/presentation/pages/chat/chat_list/views/chat_list_page_view.dart

```diff
@@ -127,6 +128,9 @@ class ChatListPageView extends StatelessWidget {
           ),
         ),
       ),
+      floatingActionButton: CreateChatButton(
+        onTap: () => null,
+      ),
```

Comment:
Should this navigate or do any logic here?

Reply:
Ah, redundant props on my side :v thanks 👍